examples/fillstates.py

```diff
@@ -2,18 +2,27 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from mpl_toolkits.basemap import Basemap as Basemap
-from matplotlib.colors import rgb2hex
+from matplotlib.colors import rgb2hex, Normalize
 from matplotlib.patches import Polygon
+from matplotlib.colorbar import ColorbarBase
 
 # Lambert Conformal map of lower 48 states.
-m = Basemap(llcrnrlon=-119,llcrnrlat=22,urcrnrlon=-64,urcrnrlat=49,
+m = Basemap(llcrnrlon=-119,llcrnrlat=20,urcrnrlon=-64,urcrnrlat=49,
             projection='lcc',lat_1=33,lat_2=45,lon_0=-95)
-# draw state boundaries.
-# data from U.S Census Bureau
-# http://www.census.gov/geo/www/cob/st2000.html
-shp_info = m.readshapefile('st99_d00','states',drawbounds=True)
-# population density by state from
-# http://en.wikipedia.org/wiki/List_of_U.S._states_by_population_density
+
+# Mercator projection, for Alaska and Hawaii
+m_ = Basemap(llcrnrlon=-190,llcrnrlat=20,urcrnrlon=-143,urcrnrlat=46,
+            projection='merc',lat_ts=20)  # do not change these numbers
+
+#%% ---------   draw state boundaries  ----------------------------------------
+## data from U.S Census Bureau
+## http://www.census.gov/geo/www/cob/st2000.html
+shp_info = m.readshapefile('st99_d00','states',drawbounds=True,
```

Comment:
can this be read in only once and the drawbounds be a part of drawing the polygons?